Source/DedicatedServer/Public/ServerConsole.h

```diff
@@ -15,6 +15,24 @@ DECLARE_LOG_CATEGORY_EXTERN( LogServerConsole, Log, All );
 	#elif PLATFORM_LINUX
 		#include "Public/Linux/LinuxConsoleOutputDevice.h"
 		#define FOutputDeviceConsolePlatform FLinuxConsoleOutputDevice
+
+		struct COORD
+		{
+			int X;
+			int Y;
+
+			COORD( int Y, int X )
+			{
+				this->Y = Y;
+				this->X = X;
+			}
```

Comment:
```suggestion
			COORD( int iX, int iY )
			{
				X = iX;
				Y = iY;
			}
```
Make sure you validate the order of the parameters on the rest of the code (with or without accepting this suggestion)